src/main/java/com/github/tomakehurst/wiremock/standalone/CommandLineOptions.java

```diff
@@ -195,6 +195,19 @@ public HttpServerFactory httpServerFactory() {
         }
     }
 
+    @Override
+    public ThreadPoolFactory threadPoolFactory() {
+        try {
+            ClassLoader loader = Thread.currentThread().getContextClassLoader();
+            Class<?> cls = loader.loadClass(
```

Comment:
Why do you need to do it this way, rather than instantiating it directly?

Reply:
It can be instantiate directly, I just copied the behaviour as 
```
    @Override
    public HttpServerFactory httpServerFactory() {
        try {
            ClassLoader loader = Thread.currentThread().getContextClassLoader();
            Class<?> cls = loader.loadClass(
                    "com.github.tomakehurst.wiremock.jetty9.JettyHttpServerFactory"
            );
            return (HttpServerFactory) cls.newInstance();
        } catch (Exception e) {
            return throwUnchecked(e, null);
        }
    }
```

because maybe I was missing something.

Reply:
OK, might be worth switching to direct if that works. Less likely to get broken by e.g. a package reshuffle.